docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -0,0 +1,246 @@
+realized when you reference the core data attribute (?):
+
+.. doctest::
+
+    >>> cube = iris.load_cube(iris.sample_data_path('air_temp.pp'))
+    >>> cube.has_lazy_data()
+    True
+    >>> the_data = cube.core_data()
+    >>> cube.has_lazy_data()
+    True
+    >>> real_data = cube.data
+    >>> cube.has_lazy_data()
+    False
+
+
+Changing a Cube's Data
+----------------------
+
+There are several methods of modifying a cube's data array, each one subtly
+different from the others.
+
+Maths
+^^^^^
+
+You can use :ref:`cube maths <cube_maths>` to make in-place modifications to
+each point in a cube's existing data array.  Provided you do not directly
+reference the cube's data, the array will remain lazy:
+
+.. doctest::
+
+    >>> cube = iris.load_cube(iris.sample_data_path('air_temp.pp'))
+    >>> cube.has_lazy_data()
+    True
+    >>> cube *= 10
+    >>> cube.has_lazy_data()
+    True
+
+Copy
+^^^^
+
+You can copy a cube and assign a completely new data array to the copy. All the
+original cube's metadata will be the same as the new cube's metadata.  However,
+the new cube's data array will not be lazy if you replace it with a real array:
+
+.. doctest::
+
+    >>> import numpy as np
+    >>> data = np.zeros((73, 96))
+    >>> new_cube = cube.copy(data=data)
+    >>> new_cube.has_lazy_data()
+    False
+
+Replace
+^^^^^^^
+
+This does essentially the same thing as `cube.copy()`, except that it provides
+a safe method of doing so for the specific edge case of a lazy masked integer
+array:
+
+.. doctest::
+
+    >>> values = np.zeros((73, 96), dtype=int)
+    >>> data =np.ma.masked_values(values, 0)
+    >>> print(data)
+    [[-- -- -- ..., -- -- --]
+     [-- -- -- ..., -- -- --]
+     [-- -- -- ..., -- -- --]
+     ...,
+     [-- -- -- ..., -- -- --]
+     [-- -- -- ..., -- -- --]
+     [-- -- -- ..., -- -- --]]
+    >>> new_cube = cube.copy(data=data)
+    >>> new_cube.has_lazy_data()
+    False
+    >>> new_cube.data = new_cube.lazy_data()
+    >>> new_cube.has_lazy_data()
+    True
+
+This method is necessary as Dask is currently unable to handle masked arrays.
+Please refer to the Whitepaper for further details.
+
+
+Coordinate Arrays
+-----------------
+
+Cubes possess coordinate arrays as well as data arrays, so these also benefit
+from Dask's functionality, although there are some distinctions between how
+the different coordinate types are treated.
+
+Auxiliary coordinates can now contain lazy arrays, so they will adhere to the
+same rules and behaviour as the data arrays.  Dimension coordinates, however,
+undergo monotonicity checks which cause the arrays to be realized upon
+construction, so they can only contain real arrays.
+
+
+Dask Processing Options
+-----------------------
+Dask applies some default values to certain aspects of the parallel processing
+that it offers with Iris. It is possible to change these values and override
+the defaults by using 'dask.set_options(option)' in your script.
+
```

Comment:
worth describing exactly *what* Dask options Iris sets first, linking it directly to the account in the Dask docs.
It may also be worth explaining how Iris "steps aside" if you make your own set_options call.